pyingest/parsers/aasnova.py

```diff
@@ -0,0 +1,136 @@
+            author = "%s, %s" % (lname, fname)
+        else:
+            author = "Astrobites"
+            lname = "Astrobites"
+        # Nova record number
+        nova_entry = entry.find('guid').text
+        links['ELECTR'] = nova_entry
+        # The record number follows from the URL:
+        # e.g. https://aasnova.org/?p=10858
+        # Of course, we can parse the URL using the urllib.parse library
+        # but that seems overkill here
+        recno = nova_entry.split('p=')[-1]
+        # Attempt to find the citation in the record (for the paper
+        # discussed in this AAS Nova entry)
+        try:
+            citation = entry.find('citation').text
+        except:
+            citation = None
+        # If we found a citation, see if we can capture a DOI
+        doi = None
+        if citation:
+            dmat = re_doi.search(citation)
+            if dmat:
+                doi = dmat.group('doi').strip()
+        # Get the publication date
+        try:
+            pdate = entry.find('pubdate').text.split()
+            pubdate = " ".join(pdate[1:4])
+            mnth = mnth2num.get(pdate[2].lower(), '00')
+            year = pdate[3]
+        except:
+            raise NoPublicationDateFound("AAS Nova entry does not contain publication date")
+        # Get the journal entry
+        journal = "AAS Nova Highlight %s, id. %s" % (pubdate, recno)
+        # Get the abstract data
+        try:
+            abstract = entry.find('description').text.replace('<![CDATA[','').replace(']]>','').strip()
+            abstract = re.sub('The post.*?appeared first on AAS Nova\.','', abstract).strip()
+        except:
+            abstract = ''
+        # Sometimes the associated DOI(s) are in the 'contents' data, 
+        # in case we did not find one earlier
+        if not doi:
+            contents = entry.find('content:encoded').text
+            # We take the first DOI we find
+            try:
+                doi = re.findall(doi_pattern, contents, re.IGNORECASE)[0]
+            except:
+                doi = None
+        if doi:
+            # If we have a DOI, assign it to the ASSOCIATED link
+            links['ASSOCIATED'] = "https://dx.doi/%s"%doi
+        # Form the bibcode
+        bibcode = "%snova.pres%s%s" % (year, recno.rjust(4, '.'), lname[0])
+        # Now we have all information to send back
+        rec = {
+            'bibcode': bibcode,
+            'title': title,
+            'authors': author,
+            'properties': links,
+            'keywords': ", ".join(keywords),
+            'pubdate': "%s/%s" % (mnth, year),
+            'publication': journal,
+            'abstract': abstract.strip()
+        }
+        return rec
+
+    def parse(self, url, **kwargs):
+        atel_recs = [{}]
+        headers = {
+            'Content-type': 'text/xml',
+            'Accept': 'text/html,application/xhtml+xml,application/xml',
+            'User-agent': 'Mozilla/5.0'}
```

Comment:
Does user agent need to be Mozilla/5.0 for some reason?  If not I would change that to something custom indicating it's coming from ADS harvesting.

Reply:
I saw you changed this in parsers.atel, would you make the same change here?

Reply:
Oops. Meant to update `aasnova.py`, but I guess it's good to have both be consistent :-)